README.md

```diff
@@ -0,0 +1,61 @@
+Installation needed:
+
+NodeJS (https://nodejs.org/en/) -> includes npm (package manager)
+
+Visual Studio Code (https://code.visualstudio.com/)
+
+Follow the installation guides
+
+VS Code Extensions:
+
+ESLint
+Prettier
+Editor Config
+
+/Reload the window
```

Comment:
Line 5-15 is Visual Studio code specific. This is part of the "recommended code editor" section. It's not a hard requirement.